### PR TITLE
Drop older versions of RuboCop

### LINKED ISSUE #131: Drop RuboCop "<= 0.60.0"
We'd like to upgrade Ruby to version 2.6. Now, only RuboCop uses Ruby 2.5.

https://github.com/sider/runners/blob/15b8cc14510844c67fafe84acfe8cb60418412b3/images/rubocop/Dockerfile#L7-L16

---

> RuboCop versions less than v0.61.0 cannot use Psych.safe_load in Ruby 2.6.

--- https://github.com/sider/runners/issues/57#issuecomment-530858769

### PR DESCRIPTION
**Pull request by yykamei**
Close #131

### CODE REVIEW

**Comment by yykamei** [comment]
I deleted smoke tests which had been in `test/smokes/rubocop/gem_install` and added unit tests alternatively.

File: test/smokes/rubocop/expectations.rb [hunk 1]
```diff
@@ -121,75 +121,41 @@
...
-    {
-      message: /The 0.50.0 and older versions are deprecated. Sider will drop these versions on January 31, 2020./,
-      file: "Gemfile"
-    },
-    { message: "Style/Tab has the wrong namespace - should be Layout", file: ".rubocop.yml" }
-  ]
-)
-
-Smoke.add_test(
-  "crash",
```
Comment (yykamei) [inline test/smokes/rubocop/expectations.rb]: [This](https://github.com/bbatsov/rubocop/issues/4794) issue was fixed.

File: test/smokes/rubocop/expectations.rb [hunk 2]
```diff
@@ -304,18 +269,6 @@
   }
 )
 
-Smoke.add_test(
-  "using_option_with_incorrect_rubocop_version",
```
Comment (yykamei) [inline test/smokes/rubocop/expectations.rb]: We no longer support the "0.59.2" and older versions of RuboCop.

File: test/smokes/rubocop/expectations.rb [hunk 3]
```diff
@@ -486,3 +439,35 @@
...
+      Failed to install gems. Sider automatically installs gems according to `sideci.yml` and `Gemfile.lock`.
+      You can select the version of gems you want to install via `sideci.yml`.
+      See https://help.sider.review/getting-started/custom-configuration#gems-option
+    TEXT
+    analyzer: nil
+  }
+)
+
+Smoke.add_test(
+  "old_ruocop_is_specified",
```
Comment (yykamei) [inline test/smokes/rubocop/expectations.rb]: Unlike sider/runner_eslint, `Ruby` module only [warns](https://github.com/sider/runners/blob/20228d6da51c80378d15b8c30cf876caa7ede8c5/lib/runners/ruby.rb#L36-L42) about the older version of RuboCop. So, the result will be "success."

File: images/rubocop/Dockerfile.erb [hunk 4]
```diff
@@ -1,16 +1,5 @@
...
-#       See https://github.com/sider/runners/issues/57#issuecomment-530858769
-ENV GLOBAL_RUBY_VERSION 2.5.7
-RUN rbenv install $GLOBAL_RUBY_VERSION && \
-    rbenv global $GLOBAL_RUBY_VERSION && \
-    rbenv versions && \
-    test $(ruby -e 'puts RUBY_VERSION') = "$GLOBAL_RUBY_VERSION" && \
-    gem install bundler --version $BUNDLER1_VERSION && \
-    gem install bundler --version $BUNDLER2_VERSION && \
-    bundle -v | grep -q "$BUNDLER2_VERSION"
-
```
Comment (ybiquitous) [inline images/rubocop/Dockerfile.erb]: 🎉

File: lib/runners/processor/rubocop.rb [hunk 5]
```diff
@@ -115,7 +112,7 @@ def check_runner_config(config)
         --cache=false
         --format=json
       ]
-      opts << "--no-display-cop-names" if support_no_display_cop_names?
+      opts << "--no-display-cop-names"
```
Comment (ybiquitous) [inline lib/runners/processor/rubocop.rb]: [nitpick] What about appending it into the `opts` array definition?

Comment (yykamei) [reply lib/runners/processor/rubocop.rb]: Nice :+1:  I'll do it.

Comment (yykamei) [reply lib/runners/processor/rubocop.rb]: b502aa7 worked. I'll rebase and merge it.

**Pull request merged**